src/import-service/serverless.yml

```diff
@@ -20,6 +20,27 @@ provider:
         - "s3:*"
       Resource:
         - "arn:aws:s3:::rs-school-app-shagon1k-uploaded/*"
+    - Effect: "Allow"
+      Action: "sqs:*"
+      Resource:
+        Fn::GetAtt: [SQSQueue, Arn]
+  environment:
+    SQS_URL:
+      Ref: SQSQueue
+
+resources:
+  Resources:
+    SQSQueue:
+      Type: AWS::SQS::Queue
+      Properties:
+        QueueName: catalogItemsQueue
```

Comment:
According to the architecture presented on the lecture - SQS Queue should be a part of product-service.

![image](https://user-images.githubusercontent.com/11696900/99982501-b1335e00-2dbb-11eb-9d17-0e030b73aedd.png)